### PR TITLE
Refactor/KAN-31/redesign - first batch

### PR DESCRIPTION
**Pull request by Neptune-Dmitry**
This summarizes work done so far redesigning the app according to Figma project. Keep in mind that more changes are to come as we have more to do to consider job done.

New images and icons were added, changed header, changed fixed buttons on the right (cart and phone), changed ProductCard component and changed bottom section of home page and footer.

### CODE REVIEW

File: src/App/AppHeader/NavBar.tsx [hunk 1]
```diff
@@ -2,16 +2,17 @@ import { chakra, Flex } from '@chakra-ui/react'
...
 
 const MainNavLink = chakra(NavLink, {
   baseStyle: {
-    fontSize: 13,
+    fontSize: 16,
+    // lineHeight: 24,
     transition: 'all 0.2s',
     fontWeight: 800,
     letterSpacing: '.35px',
     // color: 'grey.100',
```
Comment (VolodymyrKholiavko) [inline src/App/AppHeader/NavBar.tsx]: remove comment

File: src/App/AppHeader/NavBar.tsx [hunk 2]
```diff
@@ -2,16 +2,17 @@ import { chakra, Flex } from '@chakra-ui/react'
...
-    color: 'grey.700',
+    color: '#343330',
 
     _hover: {
       // color: 'blue.100',
```
Comment (VolodymyrKholiavko) [inline src/App/AppHeader/NavBar.tsx]: remove comment

File: src/App/AppHeader/NavBar.tsx [hunk 3]
```diff
@@ -24,11 +25,13 @@ const MainNavLink = chakra(NavLink, {
   },
 })
 
+console.log(NAV_LINKS)
```
Comment (VolodymyrKholiavko) [inline src/App/AppHeader/NavBar.tsx]: console.log should be deleted

File: src/App/AppHeader/Place.tsx [hunk 4]
```diff
@@ -9,7 +9,7 @@ const Place = () => {
     <Flex align="center" gap={2}>
       <Image src={point} />
       {/* <Text fontSize={14} fontWeight={600} color="grey.100"> */}
```
Comment (VolodymyrKholiavko) [inline src/App/AppHeader/Place.tsx]: remove comment please

File: src/modules/Home/components/GratitudeNote.tsx [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+        <Text color="cyanBlue.800" fontSize={28} fontWeight={700}>
+          Dziękujemy za wybór Neptunes Sushi Delivery!
+        </Text>
+
+        <Text color="cyanBlue.800" fontSize={28} fontWeight={700}>
+          Nasze sushi zostały przygotowane według przepisu samego króla mórz
+          Neptuna, co oznacza, że zapewniamy ci:
+        </Text>
+
+        <UnorderedList>
```
Comment (VolodymyrKholiavko) [inline src/modules/Home/components/GratitudeNote.tsx:31]: ListItem should be divided to component which will imports and realized with map() instead nested tags

File: src/modules/Home/components/ProductCard.tsx [hunk 6]
```diff
@@ -3,11 +3,12 @@ import { Button, Flex, Image, Text } from '@chakra-ui/react'
 import { useNavigate } from 'react-router-dom'
 import { useCallback, useEffect, useMemo, useState } from 'react'
 import stubImg from 'assets/img/stub.jpg'
+import basket from "assets/icons/basket.svg"
 import {
   useBasketContext,
   useBasketDispatchContext,
 } from 'contexts/BasketContext'
-import CountButton from 'ui/CountButton'
+// import CountButton from 'ui/CountButton'
```
Comment (VolodymyrKholiavko) [inline src/modules/Home/components/ProductCard.tsx]: remove unsuded import please

File: src/modules/Home/components/ProductCard.tsx [hunk 7]
```diff
@@ -18,7 +19,7 @@ const ProductCard = ({ product }: Props) => {
     addProduct,
     isProductAdded,
     calculateDiscountedPrice,
-    removeProduct,
+    // removeProduct,
```
Comment (VolodymyrKholiavko) [inline src/modules/Home/components/ProductCard.tsx]: remove import please

File: src/modules/Home/components/ProductCard.tsx [hunk 8]
```diff
@@ -52,21 +53,21 @@ const ProductCard = ({ product }: Props) => {
...
-  }
-
-  const handleDecrement = () => {
-    if (quantity && quantity > 1) {
-      removeProduct(product)
-    } else if (count > 1) {
-      setCount((prevCount) => prevCount - 1)
-    }
-  }
+  // const handleIncrement = () => {
```
Comment (VolodymyrKholiavko) [inline src/modules/Home/components/ProductCard.tsx]: resolve or remove unused component

File: src/modules/Home/components/ProductCard.tsx [hunk 9]
```diff
@@ -52,21 +53,21 @@ const ProductCard = ({ product }: Props) => {
...
+  //   if (quantity) {
+  //     addProduct(product)
+  //   } else {
+  //     setCount((prevCount) => prevCount + 1)
+  //   }
+  // }
+
+  // const handleDecrement = () => {
```
Comment (VolodymyrKholiavko) [inline src/modules/Home/components/ProductCard.tsx]: resolve or remove unused component

**Review by VolodymyrKholiavko — CHANGES_REQUESTED** [review]
all good, just small clear fixes

File: src/modules/Home/components/GratitudeNote.tsx (hunk 5 above)
Comment (Neptune-Dmitry) [reply src/modules/Home/components/GratitudeNote.tsx:31]: Then that means I need to create an array of strings, containing the text. However this must be corrected later when we implement translation in multiple languages. Do I proceed with the array idea?

File: src/modules/Home/components/ProductCard.tsx (hunk 9 above)
Comment (Yura-Kost) [reply src/modules/Home/components/ProductCard.tsx]: I have rid of extra extra comments

File: src/App/AppContainer.tsx [hunk 10]
```diff
@@ -25,14 +24,14 @@ const AppContainer = () => {
             bottom={{ base: 0, lg: 'none' }}
             overflowX="scroll"
           >
-            <AppSidebar />
+            {/* <AppSidebar /> */}
```
Comment (VolodymyrKholiavko) [inline src/App/AppContainer.tsx]: remove comment

File: src/App/AppHeader/AppHeaderLg.tsx [hunk 11]
```diff
@@ -1,59 +1,34 @@
...
     <Flex
       align="center"
-      // pt={2}
-      pt={4}
-      // pb={2}
-      px={2.5}
       alignItems="center"
-      justifyContent="center"
-      gap={{ base: 5, xl: 16 }}
+      // justifyContent="center"
```
Comment (VolodymyrKholiavko) [inline src/App/AppHeader/AppHeaderLg.tsx]: please, remove comment

File: src/App/AppHeader/AppHeaderLg.tsx [hunk 12]
```diff
@@ -1,59 +1,34 @@
...
+      w="100%"
+      h="64px"
+      px="75px"
+      py="8px"
+      gap="32px"
     >
-      <Flex>
-        <Logo />
-      </Flex>
+      {/* <Box position={"fixed"}  left={10}> */}
```
Comment (VolodymyrKholiavko) [inline src/App/AppHeader/AppHeaderLg.tsx]: comment

File: src/components/Logo.tsx [hunk 13]
```diff
@@ -1,24 +1,34 @@
...
-          h={{ base: '67px', xl: 'auto' }}
-        />
-        {/* <Image src={companyLogo} w={{ base: "60%", xl: 'auto' }} /> */}
+    <Link to="/" as={RouterLink} h="100%" _hover="none">
+      <Flex  alignItems={"center"}>
         <Image
           src={companyLogo}
-          w={{ base: '96px', xl: 'auto' }}
-          // h={{ base: '88px', xl: 'auto' }}
+          // w={{ base: '55px', xl: 'auto' }}
```
Comment (VolodymyrKholiavko) [inline src/components/Logo.tsx]: remove commeent please

File: src/modules/Home/components/HomeContent.tsx [hunk 14]
```diff
@@ -31,28 +31,36 @@ const HomeContent = () => {
...
+      <Container maxW="container.lg" w="85%">
+        <HomeSlider />
+        {isLoading ? (
+          <Center h={400}>
+            <Spinner />
+          </Center>
+        ) : (
+          <Flex
+            maxW="container.lg"
+            // w="85%"
```
Comment (VolodymyrKholiavko) [inline src/modules/Home/components/HomeContent.tsx]: remove comment please

File: src/modules/ScrollToTop/ui/ScrollToTop.tsx [hunk 15]
```diff
@@ -0,0 +1,50 @@
+import { ArrowUpIcon } from '@chakra-ui/icons'
+import { Box, Center } from '@chakra-ui/react'
+import { useEffect, useState } from 'react'
+
+const ScrollToTopButton = () => {
+  const [visible, setVisible] = useState(false)
+
+  const toggleVisibility = () => {
+    if (window.scrollY > 300) {
```
Comment (VolodymyrKholiavko) [inline src/modules/ScrollToTop/ui/ScrollToTop.tsx]: use useRef() for made this task instead native DOM manipulations

File: src/modules/ScrollToTop/ui/ScrollToTop.tsx [hunk 16]
```diff
@@ -0,0 +1,50 @@
...
+      setVisible(true)
+    } else {
+      setVisible(false)
+    }
+  }
+
+  const onScrollToTop = () => {
+    window.scrollTo({
```
Comment (VolodymyrKholiavko) [inline src/modules/ScrollToTop/ui/ScrollToTop.tsx]: use useRef() for made this task instead native DOM manipulations

File: src/modules/ScrollToTop/ui/ScrollToTop.tsx [hunk 17]
```diff
@@ -0,0 +1,50 @@
...
+      top: 0,
+      behavior: 'smooth',
+    })
+  }
+
+  useEffect(() => {
+    window.addEventListener('scroll', toggleVisibility)
```
Comment (VolodymyrKholiavko) [inline src/modules/ScrollToTop/ui/ScrollToTop.tsx]: use useRef() for made this task instead native DOM manipulations

File: src/modules/ScrollToTop/ui/ScrollToTop.tsx [hunk 18]
```diff
@@ -0,0 +1,50 @@
...
+    return () => {
+      window.removeEventListener('scroll', toggleVisibility)
```
Comment (VolodymyrKholiavko) [inline src/modules/ScrollToTop/ui/ScrollToTop.tsx]: use useRef() for made this task instead native DOM manipulations

**Review by VolodymyrKholiavko — CHANGES_REQUESTED** [review]
good job at all

**Pull request merged**